api_app/analyzers_manager/models.py

```diff
@@ -0,0 +1,128 @@
+from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+from cache_memoize import cache_memoize
+
+from intel_owl.secrets import get_secret
+
+
+class Analyzer(models.Model):
+    TYPE_CHOICES = (
+        ("file_analyzer", "file_analyzer"),
+        ("observable_analyzer", "observable_analyzer"),
+    )
+    QUEUE_CHOICES = (
+        ("default", "default"),
+        ("long", "long"),
+        ("local", "local"),
+    )
+    HASH_CHOICES = (("md5", "md5"), ("sha256", "sha256"))
+
+    name = models.CharField(max_length=128, blank=False, null=False, unique=True)
```

Comment:
can be skipped: `blank` and `null` are `False` by default

Reply:
Please omit this everywhere!

Reply:
I have changed where I have used `blank` and `null` based on some documentation I read. There were places it was not good of me to use them, so removed from there. But otherwise, i would prefer to have `blank` at other places for the sake of being a bit explicit.